docs/sphinx/examples/python/tutorials/Shors.ipynb

```diff
@@ -829,30 +839,31 @@
      "text": [
       "Trying a = 4\n",
       "Measurement results:\n",
-      "{ 000:5171 010:874 111:254 001:205 011:3670 100:481 101:3362 110:983 }\n",
+      "{ 000:5165 010:916 111:219 100:469 110:934 001:215 101:3595 011:3487 }\n",
       "\n",
-      "Trying nonzero bitpattern from the phase estimation: 011 = 3\n",
-      "0.375\n",
-      "[0, 2, 1, 2]\n",
-      "[(0, 1), (1, 2), (1, 3), (3, 8)]\n",
+      "Trying nonzero bitpattern from the phase estimation: 101 = 5\n",
+      "0.625\n",
+      "[0, 1, 1, 1, 2]\n",
+      "[(0, 1), (1, 1), (1, 2), (2, 3), (5, 8)]\n",
+      "testing order 1\n",
       "testing order 1\n",
       "testing order 2\n",
       "testing order 3\n",
       "Found order: 3\n",
       "The order of a is 3\n",
       "No non-trivial factor found\n",
       "retrying...\n",
-      "Trying a = 18\n",
-      "Found factor by chance: 3 7\n"
+      "Trying a = 0\n",
+      "Found factor by chance: 21 1\n"
```

Comment:
`21` and `1` are trivial factors. I don't think we want the main docs page showing those trivial factors. (I.e. I think we would prefer that it find the 3 and 7, as it was before.)

Reply:
Yeah, it must be different every time (which is unfortunate for those of us who like repeatability 😄 ). I'm just saying that it would be better if the one that we publish in the docs didn't say 21 and 1, so could you re-run it, or revert the output cells to what they were before?